packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -88,7 +119,13 @@ export const useAlertStyles_unstable = (state: AlertState): AlertState => {
   }
 
   if (state.action) {
-    state.action.className = mergeClasses(alertClassNames.action, styles.action, state.action.className);
+    // Note: inverted && actionStylesInverted.action has the highest piority and must be merged last
```

Comment:
```suggestion
```

nit: I don't think that worth a comment as it's how `mergeClasses` works

Reply:
I commented that to ensure that if this file is edited the order isn't messed up. If it is, the styles will be altered. But if you think it's unnecessary then I'll remove